core/src/main/java/org/openedx/core/CourseTab.kt

```diff
@@ -0,0 +1,9 @@
+package org.openedx.core
+
+enum class CourseTab {
+    COURSE,
+    VIDEOS,
+    DISCUSSION,
+    DATES,
+    HANDOUTS
```

Comment:
Do we need this new dependency in the Core?
I'm not sure that Core should know about course screen UI. Maybe we can replace it just with bool flags?
let's discuss it.
@HamzaIsrar12 your opinion?

Reply:
done